src/coreclr/scripts/jitrollingbuild.py

```diff
@@ -684,6 +718,11 @@ def setup_spmi_location_arg(spmi_location):
                             lambda unused: True,
                             "Unable to set git_hash")
 
+        coreclr_args.verify(args,
+                            "use_latest_jit_change",
```

Comment:
Why is this even an option? Why not do this way always? When will we ever pass `use_latest_jit_change=False`?

Reply:
When I'm testing "upload" locally, I often give it non-actual hashes, so I want that to still work.